### DISCUSSION TITLE
Accountability / Revision History / Tracking

### DISCUSSION THREAD

**Discussion by MentalGear** [ConduitPlatform/Conduit#496] [Q&A] [answered]
Thank you and Congrats on this awesome backend.

To make it really production ready for me though, I was wondering if there are modules for Account Change Tracking (Accountability) and a revision history.

Also, are Graph DB as well supported (e.g. Surreal DB) ?

**Comment by kkopanidis** [comment, marked as answer]
Greetings @MentalGear, thanks for your interest in Conduit! We don't have relevant features for this, but I don't think it would difficult for you to create such a module for your use case. Our database exposes events when modification actions happen.

We currently support MongoDB and PostgreSQL. We are working on expanding the platform to support all SQL dbs. We don't currently have a plan to support more technologies ex. Surreal, Neo4j etc, but if there is enough demand we could.
